posts/ml-sagemaker.md

```diff
@@ -0,0 +1,201 @@
+---
+authors:
+- Mallik Sambaraju
+tags:
+- Machine Learning
+- Data Science
+- AWS
+- Sagemaker
+- Python
+date: 2019-03-22T12:24:13.000Z
+title: "Linear Regression using AWS Sagemaker"
+image: 
+---
+
+Various industries such as Banking, Manufacturing, Insurance, Health, and Defence are using Machine Learning to solve many use cases such as Fraud Detection, Improving Healthcare, Personal security, Providing more secured transportation e.t.c. Recently [Ippon Technologies](https://en.ippon.tech) sent me to a summit on Data Science where I learnt about how organizations are using Machine Learning to improve their business and productivity. During the course of the summit many service and cloud providers presented various tools, libraries, algorithms and platforms for developing Machine Learning applications. One of the services introduced to me was Sagemaker by AWS. In this blog I will talk about How I implemented a basic regression model.
+
+# Machine Learning Process
+Typically Machine Learning process is an iterative process, it starts with identifying a use case to solve. Some of the steps involved in this process are as follows. This could change based on the use case you are trying to solve.
+
+* Identify and Obtain Data.
+* Pre-process and Prepare Data.
+* Train a Model using the Prepared Data.
+* Evaluate the Model.
+* Deploy the Model.
+* Obtain Feedback of the Model.
+
+Each of the above steps are iterative by themselves. Multiple iterations can happen during each of these steps depending on the data and performance of the model e.t.c.
+
+![Machine Learning Process](https://raw.githubusercontent.com/msambaraju/blog-usa/master/images/2019/03/Machine_Learning.png)
+
+# AWS Sagemaker
+AWS Sagemaker is a fully managed AWS Machine Learning service which helps in building, training and deploying Machine Learning models. It has a rich set API's, built-in algorithms, integration with various popular libraries such as Tensorflow, PyTorch, SparkML e.t.c., developers tools for authoring models, and hosted production environment for deploying the models.
+
+# Regression Model Implementation
+In this example we will develop a regression model to predict body fat percentage based on already labeled data with 14 parameters like Age, Height, Weight, Abdomen circumference e.t.c. This data is widely available on the internet. A more advanced process is to find only the relevent features or parameters to train the model.
```

Comment:
```suggestion
In this example we will develop a regression model to predict body fat percentage based on already labeled data with 14 parameters like age, height, weight, abdomen circumference etc. This data is widely available on the internet. A more advanced process is to find only the relevant features or parameters to train the model.
```
Perhaps show where you got your dataset? Not necessarily required, but if I were to follow your guide, it would be nice to have the dataset handy.